src/shrutiS/assignment23/PrimeNoCalculation.java

```diff
@@ -0,0 +1,53 @@
+/*Assignment_23 : 20th Nov'2020
+
+1) Find count of prime numbers between 2-100.
+2) find sum of all prime numbers between 2-100.
+3) find average of all prime numbers between 2-100.*/
+
+package shrutiS.assignment23;
+
+public class PrimeNoCalculation {
+	float sum;
+	int finalCount;
+
+	int findCountOfPrimeNo() {
+		for (int index = 2; index <= 100; index++) {
+			int count = 0;
+			for (int innerIndex = 1; innerIndex <= index / 2; innerIndex++) {
```

Comment:
You should start from 2 as we cannot determine 1 as prime or non-prime,
take of this next time